src/app/phast/explore-phast-opportunities/explore-phast-opportunities.component.ts

```diff
@@ -59,7 +60,11 @@ export class ExplorePhastOpportunitiesComponent implements OnInit {
   ngOnDestroy() {
     this.toastyService.clearAll();
     this.exploreOppsToast.emit(false);
-  }
+      if(this.phast.modifications[this.exploreModIndex] && !this.phast.modifications[this.exploreModIndex].phast.name){
+        this.phast.modifications[this.exploreModIndex].phast.name = 'Opportunities Modification';
+        this.emitSave.emit(true);
```

Comment:
You should use the "save" event emitter declared as an Output. This emit save won't do anything.